### PR TITLE
cpptest: add package version 1.1.2

### PR DESCRIPTION
**Pull request by mfrw**
Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Add package `cpptest` v1.1.2

###### Change Log  

- Add package

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

- Local Build with RUN_CHECK=y

BuddyBuild: 
- AMD64: [PR-4042](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=252740&view=logs&j=75cba72d-09cb-503a-3927-4ab61ac4a179&t=246631ef-4e63-576d-3b21-bc52cd4b0a93&l=1752)
- ARM64: [PR-4042](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=252739&view=results)

### CODE REVIEW

**Review by neha170 — APPROVED** [review]
Please make changes as suggested in the linter, thanks

File: SPECS-EXTENDED/cpptest/cpptest.spec [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+developing applications that use %{name}.
+
+
+%prep
+%setup -q
+#%patch0 -p1
+%build
+export CXXFLAGS="-std=c++14 $RPM_OPT_FLAGS"
+%configure --disable-static --enable-doc
+make %{?_smp_mflags} V=1
```
Comment (neha170) [inline SPECS-EXTENDED/cpptest/cpptest.spec:36]: Can we replace it with make_build ?

File: SPECS-EXTENDED/cpptest/cpptest.spec [hunk 2]
```diff
@@ -0,0 +1,154 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Name:           cpptest
+Version:        1.1.2
+Release:        20%{?dist}
+Summary:        A portable and powerful and simple unit testing framework for C++
+
+License:        LGPLv2+
+URL:            https://%{name}.sourceforge.net/
+Source0:        https://downloads.sourceforge.net/%{name}/%{name}-%{version}.tar.gz
```
[the remark this answers is not in the document]
Comment (mfrw) [reply SPECS-EXTENDED/cpptest/cpptest.spec:10]: I am not sure if i follow this suggestion.
Could you please elaborate a little?

[the remark this answers is not in the document]
Comment (neha170) [reply SPECS-EXTENDED/cpptest/cpptest.spec:10]: Sorry, my bad, please ignore this comment

[the remark this answers is not in the document]
Comment (mfrw) [reply SPECS-EXTENDED/cpptest/cpptest.spec:10]: no worries at all :)
I thought i am missing something trivial. Thank you for being vigilant @neha170

File: SPECS-EXTENDED/cpptest/cpptest.spec (hunk 1 above)
Comment (mfrw) [reply SPECS-EXTENDED/cpptest/cpptest.spec]: done. Thanks

**Pull request merged**